test/Spec.hs

```diff
@@ -10,9 +10,8 @@ module Main where
 
 import Relude
 
-import Test.Tasty (defaultMain)
-
-import Test.Relude.Property (hedgehogTestTree)
+import Test.Relude.Property (hedgehogTestList)
 
 main :: IO ()
-main = defaultMain hedgehogTestTree
+main = sequence_ hedgehogTestList
```

Comment:
I would prefer to use `checkParallel` instead of `checkSequential`. And also use the following style of tests from `membrain`:

* https://github.com/kowainik/membrain/blob/4b21b9c46ea605d7c88a0e93c98711cc9fd3d0ae/test/Spec.hs#L13-L21

These several lines are required to make tests work smoothly on Windows and on various CI services.